fnal_column_analysis_tools/arrays/Builder.py

```diff
@@ -0,0 +1,130 @@
+import awkward
```

Comment:
`from ..util import awkward`
This allows a global switch for the different awkward versions (`awkward.numba`, `awkward.cuda`, etc.)